### PR TITLE
Add `jupyter-server-proxy` support

### LINKED ISSUE #144: Add support for `jupyter-server-proxy`
Using proxy support implemented in #143, it should be easy to support `jupyter-server-proxy` for users that run Jupyter through reverse proxy.

### PR DESCRIPTION
**Pull request by randomir**
Close #144.

Support for relative path in React app pending and will be fixed with #146.

### CODE REVIEW

File: dwave/inspector/config.py [hunk 1]
```diff
@@ -38,5 +38,9 @@ def host(self):
     def base_port(self):
         return int(self._env('DWAVE_INSPECTOR_BASE_PORT', 18000))
 
+    @property
```
Comment (arcondello) [inline dwave/inspector/config.py:41]: This could also be static, not sure if that's desirable or not

**Review by arcondello — APPROVED** [review]
LGTM!

File: dwave/inspector/config.py (hunk 1 above)
Comment (randomir) [reply dwave/inspector/config.py:41]: static property?

Comment (arcondello) [reply dwave/inspector/config.py:41]: yes

Comment (arcondello) [reply dwave/inspector/config.py:41]: sorry, no. A static method. Which I suppose is why you did it this way. So ignore me.

Comment (randomir) [reply dwave/inspector/config.py:41]: given they are not natively supported, and would have to muck with metaclass, not sure if that's worth it

Comment (randomir) [reply dwave/inspector/config.py:41]: ah, didn't see your previous comment. github doesn't refresh sometimes

**Pull request merged**